elastic/kibana
# [Uptime] Make uptime settings saved object exportable

## Summary

Fix https://github.com/elastic/kibana/issues/82540

Make uptime settings saved object import/exportable

### Testing

1. Change uptime settings
2. Go to saved objects in stack management 
3. You should see the uptime saved object
4. Make sure you can export it and download/inspec it
